src/helpers/userAnalyticHelpers.js

```diff
@@ -6,17 +6,20 @@ export function segmentUA(data) {
   var analytics = (window.analytics = window.analytics);
 
   // Ensure that any UTM query parameters are sent along to Segment
-  var queryParameters = retrieveUTMQueryParameters();
-  var combinedData = Object.assign({}, data, queryParameters);
+  // var queryParameters = retrieveUTMQueryParameters();
+  // var combinedData = Object.assign({}, data, queryParameters);
+  var combinedData = stringifiedData;
```

Comment:
The data object was being passed to segment earlier. NOT the combinedData string. I suggest checking that segment is interpreting the data correctly.

Reply:
https://github.com/OlympusDAO/olympus-frontend/commit/7700e64adf438c415f2a615e3959426d01a0aa3f this commit shows where the issue is

Reply:
was using brave browser on the stake view: https://app.olympusdao.finance/#/stake